src/shared/lib/utils/common.ts

```diff
@@ -21,7 +21,7 @@ export function range(n: number): number[] {
   return [...Array(n).keys()];
 }
 
-export const isEmptyObject = (object: any) => {
+export const isObjectNotEmpty = (object: any) => {
```

Comment:
Please consider this change:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-2.html#object-type

```suggestion
export const isObjectNotEmpty = (object: object) => {
```